skymp5-server/cpp/server_guest_lib/script_classes/PapyrusSkymp.cpp

```diff
@@ -13,3 +13,32 @@ VarValue PapyrusSkymp::SetDefaultActor(VarValue self,
 
   return VarValue::None();
 }
+
+VarValue PapyrusSkymp::GetFaction(VarValue self,
```

Comment:
I don't know what's the best way to do this. let's put out of this PR for now, to make PR pass as fast as possible